src/Database/Driver/PDODriverTrait.php

```diff
@@ -94,9 +94,8 @@ public function isConnected()
                 $connected = false;
             }
         }
-        $this->connected = !empty($connected);
```

Comment:
@dereuromark I am not sure what you mean. Did you delete this line intentionally? I should have commented this line though.